docs_user/adoption-attributes.adoc

```diff
@@ -5,10 +5,14 @@
 ifeval::["{build}" == "upstream"]
 :OpenShift: OpenShift
 :osp_prev_long: OpenStack
+:osp_acro: OSP
 :OpenStackShort: OSP
 :OpenShiftShort: OCP
 :osp_curr_ver: Antelope
+:osp_prev_ver: Wallaby
 :OpenStackInstaller: TripleO
```

Comment:
this should be `OpenStackPreviousInstaller: TripleO`

Reply:
Done.